### ISSUE TITLE
Question about _target and _target_anechoic files in CEC1 datasets

### ISSUE THREAD

**Issue by Chengwei-Ouyang** [claritychallenge/clarity#3]
When I train the e009_sheffield model, both train loss and val loss (SNRLoss) will converge on 0.0043, which means all outputs for any inputs will be 0. When checking the train dataset, I find that the model using '_target_anechoic.wav' as ground truth, but the '_target_anechoic.wav' has about 20ms latency compared with both '_mix_CH1.wav' and '_target_CH1.wav'. In this way, the model is expected to predict the future information, which is impossible. If I change the ground truth from '_target_anechoic.wav' to '_target_CH1.wav', the problem is solved and the model can converge normally. Could you give me some advice on why the appoximate 20ms latency is exising between 'target_CH1.wav' and '_target_anechoic.wav' and how can I solve it? Thanks!
![image](https://user-images.githubusercontent.com/70736875/166897103-a3e8cdcd-8554-4ee5-ad4a-b62ead841286.png)

**Comment by TuZehai** [comment]
Hi,

Thank you for noticing this. I think we packed the older version anechoic signals, which is not quite correct (i.e. the old anechoic signals did not take the transmission into account, which is probably the cause of this 20ms delay). 

Here is the correct version of anechoic signals: https://mab.to/I9mkGx4wsiiaX . If you could download the 'clarity_CEC1_data.anechoic.v1_3.tgz' and replace the old ones, the training should be fine.

Thanks

**Comment by Chengwei-Ouyang** [comment]
Thanks for your reply, and I have met another problem when running the e009_sheffield model. I notice that the 'cec1_dataset.py' will get rid of the 2 second silence in the beginning of clarity data and generate 2 second audio samples randomly. But the audio files also exist 1-2 seconds silence in the end, which means the generated audio samples will include lots of silence data. In my experiments, the model can only converge when I set start equas to 0 instead of randint(wav_len - 2*44100).

**Comment by Chengwei-Ouyang** [comment]
Thanks for your help! The probelm is solved by using the new anechoic signals.
